wrapper.go

```diff
@@ -0,0 +1,88 @@
+package nmt
+
+import (
+	"crypto/sha256"
+
+	"github.com/lazyledger/nmt/namespace"
+	"github.com/lazyledger/rsmt2d"
+)
+
+var paritySharesNamespaceID = namespace.ID{0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF}
```

Comment:
Note that the tree already knows about this internally: https://github.com/lazyledger/nmt/blob/e0a317aa2d0ff84fcf1478527e4c90e6a13245d7/hasher.go#L77